src/BlazorUI/Bit.BlazorUI.Extras/Components/Map/BitMap.scss

```
@@ -0,0 +1,65 @@
+
+// OpenLayers popup. Styles live here (instead of inline cssText in the TS provider)
+// so they aren't blocked under strict CSP configurations that disallow inline styles
+// and so consumers can theme them from their own stylesheets.
+.bit-map-ol-popup {
+    position: relative;
+    max-width: 280px;
+    padding: 10px 12px;
+    background: #fff;
+    border-radius: 6px;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.25);
+    word-wrap: break-word;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)